### PR TITLE
[DOCS]: Weeks 9, 10 and 11 Blog posts

### PR DESCRIPTION
**Pull request by WassCodeur**
Weeks 9, 10 and 11 Blog posts

### CODE REVIEW

**Review by robinroy03 — APPROVED** [review]
Good work Wachiou. The videos rendered well and the content is also ok. Approving.

File: docs/source/posts/2024/2024-08-13-week9-wachiou.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+WEEK 9: Fixing Sphinx Warnings and Investigating Web Footer Issues
+==================================================================
+
+.. post:: August 13, 2024
+   :author: Wachiou BOURAIMA
+   :tags: google
+   :category: gsoc
+
+Hello everyone,
+Welcome to my Google Summer of Code (GSoC) 2024 journey! Week 9 was devoted to fixing Sphinw warnings caused by indentation errors in some docstrings of some examples in the ``auto_examples`` folder. I also started investigating why the footer of the ``FURY`` site distorts when you try to move the mouse over an element.
```
Comment (skoudoro) [inline docs/source/posts/2024/2024-08-13-week9-wachiou.rst:10]: - lowercase for `welcome`
- typo `Sphinx` and not `Sphinw`
-

File: docs/source/posts/2024/2024-08-13-week9-wachiou.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+
+Continuing the Fight Against Sphinx Warnings
+---------------------------------------------
```
Comment (skoudoro) [inline docs/source/posts/2024/2024-08-13-week9-wachiou.rst:14]: too long, not the same size as the title

**Review by skoudoro — COMMENTED** [review]
Hi @WassCodeur,

See below some comments, then it is ready to go. Thanks!

**Review by deka27 — APPROVED** [review]
Maybe for the next-week section in week 11, you can mention the preparation of the final report, optional.
Otherwise I don't see any other issues while rendering the pages.

**Review by itellaetxe — APPROVED** [review]
Hi Wachiou, good work, congrats on all the tasks you completed!
The blogs look good, all the comments I had have been already mentioned by the rest of the participants.

LGTM, approving

**Comment by WassCodeur** [comment]
> Hi Wachiou, good work, congrats on all the tasks you completed! The blogs look good, all the comments I had have been already mentioned by the rest of the participants.
> 
> LGTM, approving

Thank you @itellaetxe

**Review by skoudoro — APPROVED** [review]
all good, merging, thanks!

**Pull request merged**